hash/dc.xml

```diff
@@ -551,7 +552,7 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 		<rom name="track10.raw" size="34670832" md5="137bd822fed33a86f1d3aaf841500ffa" sha1="547aa69f3973c99608001fbffc4436e1a12d95c7"/>
 		<rom name="track11.iso" size="162893824" md5="1d4920973682a6f3260195ec975a9c05" sha1="25688c46191774b062a25ec382425f286ea22e51"/>
 		-->
-		<description>4x4 Evo (Europe, prototype 20010109)</description>
+		<description>4x4 Evo (Europe) (Proto) (20010109)</description>
```

Comment:
Leave as-is.

Reply:
Noted on not changing prototype to proto - that was just an attempt to follow a redump convention, but seems unabbreviated should be considered for Softwarelists -no issue those out as most prototypes aren't even in the redump dats anyway.